src/vs/platform/terminal/node/terminalEnvironment.ts

```diff
@@ -95,14 +95,16 @@ export function getShellIntegrationInjection(
 	if (options.shellIntegration.nonce) {
 		envMixin['VSCODE_NONCE'] = options.shellIntegration.nonce;
 	}
+	const scopedDownShellEnvs = ['PATH', 'VIRTUAL_ENV', 'HOME', 'SHELL', 'PWD'];
 	if (shellLaunchConfig.shellIntegrationEnvironmentReporting) {
+		// TODO: (Perf) Temporarily pass lists of hardcoded env vars for shell env api
```

Comment:
Isn't this describing the change here? Should `TODO` be removed and should the comment be moved to be above `scopedDownShellEnvs`?

Also nit, but `list` vs `lists` 😄

Reply:
totally! valid 100%
I took the feedback here => [0a6d37b](https://github.com/microsoft/vscode/pull/245899/commits/0a6d37b7a91d3f2b6249400f168942f4043566f8)